src/ui/ResultList/ResultList.ts

```diff
@@ -815,11 +816,12 @@ export class ResultList extends Component {
       this.options.resultContainer = $$(elemType, { className: 'coveo-result-list-container' }).el;
       this.element.appendChild(this.options.resultContainer);
     }
+    this.resultContainer = new ResultContainer(this.options.resultContainer, this.searchInterface);
   }
 
   private initWaitAnimationContainer() {
     if (!this.options.waitAnimationContainer) {
-      this.options.waitAnimationContainer = this.options.resultContainer;
+      this.options.waitAnimationContainer = this.resultContainer.resultContainerElement.el;
```

Comment:
Small nitpick, but I'd simply have an `get el` method in the `ResultContainer` class, so the code is a bit more concise here.